jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -160,37 +178,34 @@ protected void configure(QuickStartGeneratorConfiguration generator, WebAppConte
     @Override
     public void configure(WebAppContext context) throws Exception
     {
-        if (!_quickStart)
```

Comment:
I really don't want to change the logic here.

Reply:
This logic (the `_quickStart` field) breaks when there are more than 1 WebAppProvider or someone that is using parallel deployment.
The use of fields in the Configuration to track behavior between preConfigure / configure / postConfigure is a bug that needs to be fixed.

Reply:
Agreed to remove `_quickstart`, however that can be done without all the other logic changes.